data/Molcas/basicMolcas8.0/dvb_sp.out

```diff
@@ -0,0 +1,1853 @@
+                               MOL                                    
+                               MOL                                    
+                   MOL        CASMO                                   
+                  MOLCA       SM OL                                   
+                  MO LC       AS  MO                                  
+                  MO  LC     AS   MO                                  
+                  MO  L  CAS      MO                                  
+                  M  O   LCA   S  MO                                  
+                    MOL   CA   SMO  L                                 
+              MOL   CAS   M   OLCA   SM                               
+          MOLCASM  OLCAS     MOLCA   SMOLCA                           
+       MOLCASMOL   CASMOLCASMOLCAS   MOLCASMOL                        
+      MOLCASMOLC   AS            MO  LCA    SMO                       
+      MOLCASMOLC       ASMOLCA        S     MOL                       
+      MOLCASMOLC    ASMOLCASMOLCA     S     MOL                       
+      MOLCASMOLC   ASMOLCASMOLCASM    O     LCA                       
+      MOLCASMOL   CASMOLCASMOLCASMO   L     CAS    MOLC    AS     MOL 
+      MOLCASMOL   CASMOLCASMOLCASMOL  C     ASM   OL  CA   SM    O   L
+      MOLCASMOL   CASMOLCASMOLCASMO         LCA   S        M O   L    
+      MOLCASMO     LCASMOLCASMOLCA          SMO   L       C  A    SMO 
+      MOLCAS        MOLCASMOLCASM           OLC   A       SMOL       C
+      MOL     CASM     OLCASMO     L     C  ASM   O    L CA  SM O    L
+            MOLCASMOL            CASM    O         LCAS  M    O  LCAS 
+       MOLCASMOLCASMOLCASMOLCASMOLCASMOLCASMOL                        
+          MOLCASMOLCASMOLCASMOLCASMOLCASMOL                           
+              MOLCASMOLCASMOLCASMOLCASM                               
+                 MOLCASMOLCASMOLCASM        version 8.0
+                     MOLCASMOLCA                                      
+                         MOL                                          
+
+                     Copyright, all rights, reserved:                 
+                   Permission is hereby granted to use                
+          but not to reproduce or distribute any part of this         
+       program. The use is restricted to research purposes only.      
+                      Lund University, Sweden, 2014.                  
+                                                                      
+            For the author list and the recommended citation,         
+             consult http://www.molcas.org/citations.html             
+
+
+   -------------------------------------------------------------------
+  |                                                                   
+  |   Project         = dvb_sp_un
+  |   Submitted from  = /scratch/2747562.clusman0a.frank.sam.pitt.edu
+  |   Scratch area    = /scratch/2747562.clusman0a.frank.sam.pitt.edu/dvb_sp_un
```

Comment:
This output file should be called `dvb_sp_un.out`, since it's the unrestricted cation. The input file is correctly named for `dvb_sp.in`.